contracts/ethregistrar/ETHRegistrarController.sol

```diff
@@ -215,37 +210,34 @@ contract ETHRegistrarController is
         payable
         override
     {
-        _renew(name, duration, 0, 0);
+        _renew(name, duration, 0);
     }
 
     function renewWithFuses(
```

Comment:
Do we still need two versions (renew with fuse 0 and renew with fuses)? Can we now just always renew with the same fuse they had earlier?

Reply:
No, we need both. Because non-owners should not be able to burn fuses at all. So the second function did two things

1. Burn any additional fuses they want
2. Extend expiry

Now they can just do 1., however we could force them to just do a second transaction.